loyno-mathcs/partner
# WIP password change: routes and view. Not working yet.

Resolves #4  

### Description

The feature is now working and has a feature test. Let me know if there are concerns or questions.



### Type of change



* New feature (non-breaking change which adds functionality)
* This change requires a documentation update - not sure. is there user-focused documentation on login/account management that would need to be updated?

### How Has This Been Tested?


The feature test is readable and easy to understand what is being tested. The test first creates an account, goes to update the partner's password, and check if the new password is valid while making sure that the old password no longers works.

To test manually:

1. Login
2. Click "Edit Partner"
3. Scroll to the bottom and click "Change your password" (bottom left)
4. Enter the requested information and click "Update"
5. You should be redirected to the "My Organization" page and see a green box at the top that says "Your account has been successfully updated"
6. Logout, and try logging in with the new password

### Screenshots
